oneview/resource_enclosure_group.go

```diff
@@ -66,25 +64,25 @@ func resourceEnclosureGroup() *schema.Resource {
 					Type: schema.TypeString,
 				},
 				Set: schema.HashString,
```

Comment:
as discussed we should handle it differently

Reply:
Done